_locales/pl/messages.json

```
@@ -0,0 +1,387 @@
+{
+	"extension_description": {
+		"message": "Dodaje linki SteamDB i nowe funkcjonalności w sklepie i społeczności Steam. Wyświetla najniższe ceny gier i ich statystyki."
+	},
+
+	"steamdb_options": {
+		"message": "Opcje SteamDB"
+	},
+	"steamdb_calculator": {
+		"message": "Kalkulator SteamDB"
+	},
+	"steamstatus": {
+		"message": "Sprawdź steamstat.us"
+	},
+	"steamstatus_downtime": {
+		"message": "Wygląda na to, że Steam doświadcza pewnych awarii. "
+	},
+
+	"view_on_steamdb": {
+		"message": "Wyświetl na SteamDB"
+	},
+	"view_on_pcgamingwiki": {
+		"message": "Wyświetl na PCGamingWiki"
+	},
+	"view_your_achievements": {
+		"message": "Wyświetl swoje osiągnięcia"
+	},
+	"store_page": {
+		"message": "Strona sklepu"
+	},
+	"error": {
+		"message": "Błąd"
+	},
+	"search": {
+		"message": "Szukaj"
+	},
+
+	"id_app": {
+		"message": "Aplikacja $1"
+	},
+	"id_sub": {
+		"message": "Paczka $1"
+	},
+	"id_bundle": {
+		"message": "Zestaw $1"
+	},
+
+	"app_steamdb_rating": {
+		"message": "Recenzje SteamDB:"
+	},
+	"app_steamdb_rating_responsive": {
+		"message": " RECENZJE STEAMDB"
+	},
+	"app_steamdb_rating_tooltip": {
+		"message": "$1 z $2 recenzji jest pozytywne (wszystkie typy zakupów)"
+	},
+	"app_depots_update": {
+		"message": "Ostatnia aktualizacja:"
+	},
+	"app_depots_updated_short": {
+		"message": "Zaaktualizowano"
+	},
+	"app_stats_online": {
+		"message": "Statystyki online"
+	},
+	"app_stats_online_now": {
+		"message": "Online teraz:"
+	},
+	"app_stats_peak_today": {
+		"message": "24-godzinny szczyt:"
+	},
+	"app_stats_alL_time_peak": {
+		"message": "Najwyższy szczyt w historii:"
+	},
+	"app_stats_followers": {
+		"message": "Obserwujący:"
+	},
+	"app_nothing_selected": {
+		"message": "nic nie zostało wybrane"
+	},
+
+	"app_lowest_price": {
+		"message": "Najniższa zarejestrowana cena na SteamDB to <b>$1</b>"
+	},
+	"app_lowest_price_discount": {
+		"message": "Najniższa zarejestrowana cena na SteamDB to <b>$1</b> na <b>-$2%</b>"
+	},
+	"app_lowest_price_limited": {
+		"message": "Najniższa zarejestrowana cena na SteamDB to <b>$1</b>, a w ciągu ostatnich dwóch lat to <b>$2</b>"

```

```suggestion
		"message": "Najniższa zarejestrowana cena na SteamDB to <b>$1</b>, a w ciągu ostatnich dwóch lat <b>$2</b>"

```

2x powtórzenie brzmi nienaturalnie.